pkg/apis/application/v1alpha1/types_test.go

```diff
@@ -1090,9 +1090,8 @@ func TestAppProjectSpec_DestinationClusters(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			d := AppProjectSpec{Destinations: tt.destinations}
-			if got := d.DestinationClusters(); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("AppProjectSpec.DestinationClusters() = %v, want %v", got, tt.want)
-			}
+			got := d.DestinationClusters()
+			assert.Truef(t, reflect.DeepEqual(got, tt.want), "AppProjectSpec.DestinationClusters() = %v, want %v", got, tt.want)
```

Comment:
This should use require. Also, let's have a second argument assigned on a separate line for better readability.